index.html

```diff
@@ -0,0 +1,117 @@
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/8c5a2b9118.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <a class="logo" href="index.html">
+      <img src="assets/images/Logo-Placeholder.png" alt="Logo">
+    </a>
+    <nav>
+      <div class="menu-icon gray">
+        <i class="fa fa-bars"></i>
+      </div>
+      <div class="exit-icon gray hide">
+        <i class="fa fa-xmark"></i>
+      </div>
+      <div class="menu hide">
+        <ul>
+          <li>
+            <a href="about.html">About</a>
+          </li>
+          <li>
+            <a href="#">Program</a>
+          </li>
+          <li>
+            <a href="#">Join</a>
+          </li>
+          <li>
+            <a href="#">Sponsor</a>
+          </li>
+          <li>
+            <a href="#">News</a>
+          </li>
+          <li>
+            <a class="btn" href="#">Get invitation</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section id="headline">
+    <div class="headline-con">
+      <p class="orange">"Hello! You've been invited to"</p>
+      <h1>MY BIRTHDAY!, GLOBAL CELEBRATION 2023</h1>
+      <div class="description gray">
+        <p>A joyful celebration, celebrating a joyful-ish young adult who's in the process of changing his life for the better.</p>
+      </div>
+      <div class="event-details gray">
+        <span>2023.03.25(SAT) ~ 26(SUN)</span>
+        <p class="event-location">@Author's residence</p>
+      </div>
+    </div>
+  </section>
+
+  <section id="programs">
+    <h2>Main Program</h2>
+    <div class="cards-container program-container">
+      <div class="program-card">
+        <img class="svg-white" src="assets/images/Lecture.svg" alt="">
+        <h3 class="orange">Guidelines</h3>
+        <p class="white">Remember, always be mindful, and don't break a thing.</p>
+      </div>
+      <div class="program-card">
+        <img class="svg-white" src="assets/images/Exhibition.svg" alt="">
+        <h3 class="orange">Activities</h3>
+        <p class="white">Learn new games, in the activities that'll take place.</p>
+      </div>
+      <div class="program-card">
+        <img class="svg-white" src="assets/images/Forum.svg" alt="">
+        <h3 class="orange">Meet new faces</h3>
+        <p class="white">Since this is the global event, everyone will be invited.</p>
+      </div>
+      <div class="program-card">
+        <img class="svg-white" src="assets/images/Workshop.svg" alt="">
+        <h3 class="orange">Preparations</h3>
+        <p class="white">You can join virtually or physically, just know that the space is limited.</p>
+      </div>
+      <div class="program-card">
+        <img class="svg-white" src="assets/images/Network.svg" alt="">
+        <h3 class="orange">Projects</h3>
+        <p class="white">If you're interested in my works, feel free to explore the site.</p>
```

Comment:
- [ ] `alt` attributes are important in an image, the alt attribute provides alternative information for an image if a user for some reason cannot view it. Kindly add a description for all the images in your alt attribute.